README.md

```diff
@@ -44,8 +44,52 @@ You can find the client for the Fronts tool in [fronts-client](/fronts-client).
 
-#### Troubleshooting
-##### Postgres
+
+## Different tools in this codebase
+
+### The Fronts Tool
+The most important part of this app is the Fronts Tool, used to curate the web and app front pages of the Guardian. Additionally it is used to manually curate emails.
+
+In the UI, users can move stories from the Content API feed into different front pages or emails. These fronts can then be launched or published.
+
+The Fronts Tool uses the most up-to-date front end, a React-Redux app located in [fronts-client](https://github.com/guardian/facia-tool/tree/master/fronts-client)
+
+Useful things to know:
+* An api call to Ophan pulls in page view data to allow users to understand at a glance that story's performance in the context of the front it is in
+
+* Containers can be shared between fronts. An update to a container in one place updates it everywhere
+
+* Certain containers have geolocation properties. This means they can only be seen by users in the location specified
+
+* Interactive atoms can be pasted into a front as CAPI links. This allows designers and the interactives team to create special content eg banners showing election results. Apart from articles, interactive atoms are the only other type of CAPI content allowed on Fronts
+
+* In the Email editor only - free text boxes can be created. This allows email editors to add introductions to their emails by hijacking the headline field. Rich text editing is enabled for this using Prosemirror.
+
+
+### The Editions creator
+
+The Editions interface is used to curate content on the Editions app (currently known as The Daily on iOS and Android). Editions also uses the fronts-client front end, and can be accessed from the Manage Editions menu on the [homepage](https://fronts.code.dev-gutools.co.uk/v2).
+
+Curation works in the same way as the main fronts tool. But there are these differences:
+
+* Fronts are part of Issues (normally one issue per day). You need to create or open a valid issue to make changes
+
+* Editions are published once instead of being continually updated and launched
+
+
+### The Config Tool
+
+Fronts and collections are configured in the config tool. This allows for the name, geolocation and visual properties of collections to be set. The types and order of collections in a Front is set here too. This tool uses the "old" front end client. Each "type of fronts" has its own config url eg: https://fronts.gutools.co.uk/email/config
+
+
+### The Breaking News Tool
+
+This enables breaking news notifications to be sent to app users who have signed up. This tool also uses the "old" front end client. You need permission from
+Central Production to access this tool. It does not exist in the CODE environment.  https://fronts.gutools.co.uk/breaking-news
+
```

Comment:
Can anyone @jonathonherbert  @akash1810 shed more light on the workings of the breaking news app? Is it true it doesn't exist in CODE (I couldn't see it, that's why I wrote this.) What else do people need to know about it?

Reply:
Discussed IRL 👍